src/cf-core/src/cf-base.less

```diff
@@ -291,6 +291,14 @@ h4,
     .h6 + & {
         margin-top: unit( 30px / @font-size, em );
     }
+
+    .respond-to-max( @bp-xs-max, {
+        @h4-font-size-on-xs: 16px;
+
+        margin-bottom: unit( 10 / @h4-font-size-on-xs, em );
```

Comment:
Should `10` have a unit type?

Reply:
It's apparently not necessary, but if our convention is to include one, I'll add it. I admit I didn't look elsewhere for prior art.

Reply:
It does seem that we usually include the `px` most of the time.